### PR TITLE
Development

### PR DESCRIPTION
**Pull request by timoteipalade**

### CODE REVIEW

File: Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift [hunk 1]
```diff
@@ -16,6 +16,15 @@ class AdBlockerPanel: AntitrackingPanel {
         if let label = toTableViewButton.subviews.first as? UILabel {
             label.text = NSLocalizedString("AdBlocking Information", tableName: "Cliqz", comment: "AdBlocking Information text for landscape mode.")
         }
+        
+        if let companiesLabel = self.legendView.subviews[1] as? UILabel where companiesLabel.tag == 10{
+            companiesLabel.text = NSLocalizedString("Companies title adblocking", tableName: "Cliqz", comment: "AdBlocking UI title for companies column")
```
Comment (mahmoud-adam85) [inline Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift]: follow the convention of using the English label as key
i.e.
`companiesLabel.text = NSLocalizedString("Companies", tableName: "Cliqz", comment: "AdBlocking UI title for companies column") `

File: Client/Info.plist [hunk 2]
```diff
@@ -73,6 +73,8 @@
 	<string>Used if user wants to send video feedback or bug report</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Websites you visit may request your location.</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>CLIQZ will only use this when you choose to record videos inside the application.</string>
```
Comment (mahmoud-adam85) [inline Client/Info.plist]: you need to localize this message in `inforPlist.strings`

File: Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift [hunk 3]
```diff
@@ -16,6 +16,15 @@ class AdBlockerPanel: AntitrackingPanel {
         if let label = toTableViewButton.subviews.first as? UILabel {
             label.text = NSLocalizedString("AdBlocking Information", tableName: "Cliqz", comment: "AdBlocking Information text for landscape mode.")
         }
+        
+        if let companiesLabel = self.legendView.subviews[1] as? UILabel where companiesLabel.tag == 10{
```
Comment (naira-cliqz) [inline Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift]: Space... :)

File: Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift [hunk 4]
```diff
@@ -16,6 +16,15 @@ class AdBlockerPanel: AntitrackingPanel {
...
+            companiesLabel.text = NSLocalizedString("Companies title adblocking", tableName: "Cliqz", comment: "AdBlocking UI title for companies column")
+        }
+        
+        if let countLabel = self.legendView.subviews[2] as? UILabel where countLabel.tag == 20{
```
Comment (naira-cliqz) [inline Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift]: Spaceeee

File: Client/Cliqz/Foundation/JSEngine/InterceptorURLProtocol.swift [hunk 5]
```diff
@@ -16,6 +16,11 @@ class InterceptorURLProtocol: NSURLProtocol {
     
     //MARK: - NSURLProtocol handling
     override class func canInitWithRequest(request: NSURLRequest) -> Bool {
+
+        if request.URL?.absoluteString == "http:/"{
```
Comment (naira-cliqz) [inline Client/Cliqz/Foundation/JSEngine/InterceptorURLProtocol.swift]: One more space :D

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 6]
```diff
@@ -1481,8 +1481,12 @@ class BrowserViewController: UIViewController {
         }
     }
     func goForward(){
-        if(tabManager.selectedTab?.canGoForward == true && homePanelController == nil){
+        if(tabManager.selectedTab?.canGoForward == true){// && homePanelController == nil){
```
Comment (naira-cliqz) [inline Client/Frontend/Browser/BrowserViewController.swift]: Space 😸

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 7]
```diff
@@ -1481,8 +1481,12 @@ class BrowserViewController: UIViewController {
...
             tabManager.selectedTab?.goForward()
+            if (homePanelController != nil){
```
Comment (naira-cliqz) [inline Client/Frontend/Browser/BrowserViewController.swift]: 😸

File: Client/Application/AppDelegate.swift [hunk 8]
```diff
@@ -297,6 +297,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         AntiTrackingModule.sharedInstance.initModule()
         AdblockingModule.sharedInstance.initModule()
         
+        //check if application was updated
+        
+        let standardDefaults = NSUserDefaults.standardUserDefaults()
+        if let currentVersion = NSBundle.mainBundle().infoDictionary?["CFBundleShortVersionString"] as? NSString{
+            if let savedVersion = standardDefaults.objectForKey("lastVersion") as? NSString where savedVersion != currentVersion{
```
Comment (naira-cliqz) [inline Client/Application/AppDelegate.swift]: 😸

**Review by naira-cliqz — DISMISSED** [review]

**Comment by timoteipalade** [comment]
Comments solved

**Pull request merged**